docs/Parameter.rst

```diff
@@ -0,0 +1,72 @@
+Parameter Space
+================================================
```

Comment:
```suggestion
Parameter Space
=============
```
You should actually get a warning in the Sphinx build for this.
The symbols under the header should always be exactly as long as the heading itself.